crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -145,12 +224,18 @@ impl DiskFileSystem {
         }
 
         watcher_guard.replace(watcher);
+        drop(watcher_guard);
+
+        let disk_watcher = self.watcher.clone();
+        let root_path = self.root_path().to_path_buf();
```

Comment:
Ditto unused. Could we make a consistent interface between Mac and Linux so we can just make calls to empty methods?